code/__DEFINES/colors.dm

```diff
@@ -184,3 +184,4 @@
 #define COLOR_ROCKPLANET_LIGHT "#db9e9f"
 #define COLOR_SANDPLANET_LIGHT "#EEEEEE" //EEEEEEEEEE
 #define COLOR_WASTEPLANET_LIGHT "#FA644B"
+#define COLOR_FOGGY_LIGHT "#849abf"
```

Comment:
```suggestion
#define COLOR_BATTLEFIELD_PLANET_LIGHT "#849abf"
```

Reply:
I do want to generalize it in case a mapper wants to use it for their punchcard

Reply:
Or admin uses it for an event